### PR TITLE
trivial: fix c-parser link

### PR DESCRIPTION
**Pull request by corlewis**
This reverts commit 985ce0d28a87298427e9836055cfbad0751b13c5.

### CODE REVIEW

**Review by lsf37 — APPROVED** [review]
👍 (you'll have to fix up the sign-off check before it lets you merge)

**Comment by corlewis** [comment]
> 
> 
> 👍 (you'll have to fix up the sign-off check before it lets you merge)

Whoops, good point!

**Pull request merged**
